doc/prop.xml

```diff
@@ -852,3 +852,42 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IsHamiltonianDigraph">
+<ManSection>
+  <Prop Name="IsHamiltonianDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    If digraph is Hamiltonian, this property returns
```

Comment:
`A` tags around `digraph`, since it's the argument. Also have the word `then` after the comma: `, then this...`